components/atom/tag/src/index.js

```diff
@@ -9,15 +9,20 @@ const ACTIONABLE_ONLY_PROPS = [
   'iconPlacement',
   'target',
   'actionable',
-  'linkFactory'
+  'linkFactory',
+  'rel'
 ]
 const STANDARD_ONLY_PROPS = ['closeIcon', 'onClose']
 const SIZES = {
   LARGE: 'large',
   MEDIUM: 'medium',
   SMALL: 'small'
 }
-
+const LINK_TYPES = {
```

Comment:
The idea is to use these constants even in the prop types definition.

Reply:
Changed